scripts/scripts.js

```diff
@@ -71,6 +71,169 @@ export function createTag(tag, attributes, html) {
   return el;
 }
 
+export function showSnackbar(message, typeClass) {
+  const snackbar = document.createElement('div');
+  snackbar.classList.add('snackbar', typeClass);
+  snackbar.textContent = message;
+
+  document.body.appendChild(snackbar);
+
+  setTimeout(() => {
+    snackbar.classList.add('show');
+    setTimeout(() => {
+      snackbar.classList.remove('show');
+      snackbar.remove();
+    }, 2500);
+  }, 100);
+}
+
+export function createForm(config) {
```

Comment:
I recommend implementing the modal dialogs using https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

This makes compliance with accessibility much easier, and handling of backdrop clicks and escape key are much easier. Also this dialog is missing an option to close. 

![Screenshot 2023-06-06 at 15 00 47](https://github.com/hlxsites/zemax/assets/2250964/e8ca7b80-2597-496e-b787-3c6baa2c7eb0)

Reply:
Here is a simple dialog that I created: 

https://github.com/hlxsites/zemax/blob/83bef57fd5c80a247aef05a492087f763b9ddcac/blocks/header/header.js#L67-L79

How to open and close: 
https://github.com/hlxsites/zemax/blob/83bef57fd5c80a247aef05a492087f763b9ddcac/blocks/header/header.js#L341-L346

And how to handle click on backdrop: 
https://github.com/hlxsites/zemax/blob/83bef57fd5c80a247aef05a492087f763b9ddcac/blocks/header/header.js#L347-L354

And backdrop style: 
https://github.com/hlxsites/zemax/blob/83bef57fd5c80a247aef05a492087f763b9ddcac/blocks/header/header.css#L378-L380

Reply:
I was able to do most of the part of this, for back drop click I would need some CSS help I did tried to incorporate what. I raised separate bug to handle that tomorrow. https://github.com/hlxsites/zemax/issues/197 
I would like to get this PR out soon so rest of the testing on profile page can continue

Reply:
Some of the forms the one in screenshot are same in current production environment but I have fixed the close icon on top that will give the user option to close the modal